google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -1532,6 +1533,38 @@ public void testCreateAndGetTable() {
     assertTrue(remoteTable.delete());
   }
 
+  @Test
+  public void testCreateAndGetTableWithBasicTableMetadataView() {
+    String tableName = "test_create_and_get_table_with_basic_metadata_view";
+    TableId tableId = TableId.of(DATASET, tableName);
+    TimePartitioning partitioning = TimePartitioning.of(Type.DAY);
+    Clustering clustering =
+        Clustering.newBuilder().setFields(ImmutableList.of(STRING_FIELD_SCHEMA.getName())).build();
+    StandardTableDefinition tableDefinition =
+        StandardTableDefinition.newBuilder()
+            .setSchema(TABLE_SCHEMA)
+            .setTimePartitioning(partitioning)
+            .setClustering(clustering)
+            .build();
+    Table createdTable = bigquery.create(TableInfo.of(tableId, tableDefinition));
+    assertNotNull(createdTable);
+    assertEquals(DATASET, createdTable.getTableId().getDataset());
+    assertEquals(tableName, createdTable.getTableId().getTable());
+    TableOption tableOption = BigQuery.TableOption.tableMetadataView(TableMetadataView.BASIC);
```

Comment:
Are BASIC and STORAGE_STATS, the only supported TableMetadataViews?
If not, can we add tests to verify all the values in the enum works well with the library?

Reply:
nope, added the tests below based off the discovery doc for the other params